lib/rules/prefix-members.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview Enforce "_" prefix to user methods in a React component definition
+ * @author Adrian Toncean
+ */
+'use strict';
+
+var Components = require('../util/Components');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = Components.detect(function(context, components, utils) {
```

Comment:
This should use the [new eslint rule format](http://eslint.org/blog/2016/07/eslint-new-rule-format)